app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: dict) -> list:
+    for per in people:
+        Person(per["name"], per["age"])
+
+    for per in people:
```

Comment:
per is not a good name, name it somehow more explicit, f.e. if we already have people, name it person, of single_person etc